Makefile

```diff
@@ -15,13 +15,14 @@ tidy:
 
 GOLANGCI_LINT = $(shell pwd)/bin/golangci-lint
 golangci-lint:
+	rm -rf bin || :
 	@[ -f $(GOLANGCI_LINT) ] || { \
```

Comment:
This file exists check can be removed then if we are removing the dir/file anyway at start.

Reply:
Took care of it! Thanks